File: desktop/src/BV87.Core/Models/DesktopLoginResponse.cs
namespace BV87.Core.Models;

public sealed class DesktopLoginResponse
{
    public string AccessToken { get; set; } = string.Empty;
    public string RefreshToken { get; set; } = string.Empty;
    public string TokenType { get; set; } = "Bearer";
    public long ExpiresInSeconds { get; set; }
    public UserProfile User { get; set; } = new();
}

File: backend/src/main/java/com/bv87/diemdanh/dto/DesktopLoginResponse.java
package com.bv87.diemdanh.dto;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class DesktopLoginResponse {
    private final String accessToken;
    private final String refreshToken;
    private final String tokenType;
    private final long expiresInSeconds;
    private final LoginResponse user;
}
